lib/bulbs-elements/util/prep-reading-list-analytics.js

```diff
@@ -0,0 +1,19 @@
+import prepGaWrapper from './prep-ga-wrapper';
+import getAnalyticsManager from './get-analytics-manager';
+
+export default function prepReadingListAnalytics (element, dimension12) {
+  let parent = element.closest('bulbs-reading-list-item')[0];
+
+  if (parent) {
+    this.parentAnalyticsDimensions = parent.dataset.contentAnalyticsDimensions;
+    this.href = parent.dataset.href;
+    this.title = parent.dataset.title;
+  }
+
+  let dimensionObject = {
+    dimension12: dimension12,
+  };
+
+  this.analyticsManager = getAnalyticsManager();
+  this.analyticsWrapper = prepGaWrapper(this.parentAnalyticsDimensions, dimensionObject);
```

Comment:
I'm with Chris here. 

I think also in general calling a utility function with a bound `this` rather than having a regular named argument is a little confusing.

`function foo (arg2, arg3)` vs `function foo (arg1, arg2, arg3)`